ert_shared/plugins/hook_specifications/logging.py

```diff
@@ -4,9 +4,9 @@
 @hook_specification
 def add_log_handle(logging):
     """
-    This hook allows the user to add log hooks and change formatting
+    Modify ert's logging instance, like adding handlers, changing formatting etc.
```

Comment:
Maybe remove `etc` and write `for example adding handlers and changing formatting`?

Reply:
Agree.
I've made changes, squashed and rebased.